@vates/coalesce-calls/index.test.js

```diff
@@ -1,15 +1,18 @@
 'use strict'
 
-/* eslint-env jest */
-
-const { coalesceCalls } = require('./')
+// eslint-disable-next-line n/no-unpublished-require
+const { describe, it } = require('test')
+const assert = require('assert')
+const { coalesceCalls } = require('./index')
 
 const pDefer = () => {
   const r = {}
+
   r.promise = new Promise((resolve, reject) => {
-    r.reject = reject
     r.resolve = resolve
+    r.reject = reject
   })
+
```

Comment:
These changes are unnecessary, you can remove them to keep the commit as focus as possible :slightly_smiling_face:

Reply:
You're right, I'll correct that.